connector/connect/server/src/main/scala/org/apache/spark/sql/connect/service/SessionHolder.scala

```diff
@@ -248,15 +249,15 @@ case class SessionHolder(userId: String, sessionId: String, session: SparkSessio
   private[connect] def updateAccessTime(): Unit = {
     lastAccessTimeMs = System.currentTimeMillis()
     logInfo(
-      log"Session ${MDC(SESSION_KEY, key)} accessed, " +
-        log"time ${MDC(LAST_ACCESS_TIME, lastAccessTimeMs)} ms.")
+      log"Session ${MDC(LogKeys.SESSION_KEY, key)} accessed, " +
```

Comment:
The key contains user id and session id. Let's follow the style of the other entry:
```
session with userId: ${MDC(LogKeys.USER_ID, userId)} and " +
      log"sessionId: ${MDC(LogKeys.SESSION_ID, sessionId)
```